testing/trino-testing/src/main/java/io/trino/testing/AbstractTestingTrinoClient.java

```diff
@@ -173,7 +175,8 @@ private static ClientSession toClientSession(Session session, URI server, Durati
     private static Map<String, ClientSelectedRole> getRoles(Session session)
     {
         ImmutableMap.Builder<String, ClientSelectedRole> builder = ImmutableMap.builder();
-        session.getIdentity().getConnectorRoles().forEach((key, value) -> builder.put(key, toClientSelectedRole(value)));
+        session.getIdentity().getEnabledRoles().forEach(role -> builder.put("system", toClientSelectedRole(new SelectedRole(ROLE, Optional.of(role)))));
```

Comment:
Why `system`? Isn't this will fail if there are multiple enabled roles?

Reply:
We use "system" catalog as a placeholder for system roles. See for example `io.trino.execution.SetRoleTask#execute`

Reply:
Got it! Thanks for pointing.

Also, one more doubt can there be multiple enabled roles? In that case, I think immutableMap builder will throw an error

Reply:
I don't think so. I would like to change it to single value but I don't want to change the API in this PR.